### DISCUSSION TITLE
Public database plans?

### DISCUSSION THREAD

**Discussion by dtbuchholz** [melih-unsal/DemoGPT#32] [General]
Hey y’all—I noticed you have future plans for implementing a public database in this workflow. 

What data/types will be stored there? You noted “it’ll be used for retrieving similar examples during the refining process,” so I’m just curious what that data actually looks like from a db perspective (e.g., floating point vectors, structured/unstructured, etc.).

Related—what type of database do you think you’ll use (vector, relational, k-v, etc.)?

(Note: I’m actually building a public database network and am just looking for directional insight on what builders need in this space. Any feedback would be greatly appreciated!)

**Comment by melih-unsal** [comment]
Hı @dtbuchholz , 
We will put similar examples to feed the corresponding models for each step. It will be a vector database holding the embeddings of succesfull examples for each step. These examples will be just strings.

**Comment by dtbuchholz** [comment]
cool, makes sense! i'll go ahead and close this out, appreciate the input.
